packages/jimp/README.md

```diff
@@ -77,21 +77,6 @@ If you're using a web bundles (webpack, rollup, parcel) you can benefit from usi
 import Jimp from "jimp/es";
 ```
 
-### WebPack
-
-If you're using webpack you can set `process.browser` to true and your build of jimp will exclude certain parts, making it load faster.
```

Comment:
This isn't a problem anymore and was always a misunderstanding on how to make a browser bundle 😓